data/food/export_agb/ingredients_base.json

```diff
@@ -461,39 +460,18 @@
     "raw_to_cooked_ratio": 0.856,
     "density": 0.6195,
     "transport_cooling": "none",
-    "visible": true,
-    "variants": {
-      "organic": {
-        "ratio": 0.0,
-        "beyondLCA": {
-          "agro-diversity": 0.0,
-          "agro-ecology": 0.0,
-          "animal-welfare": 0.0
-        }
-      }
-    }
+    "visible": true
   },
   {
     "id": "eggplant",
     "name": "Aubergine",
-    "default": "Zucchini, conventional, national average, at farm gate {FR} U",
+    "default": "In-season eggplant consumption mix",
```

Comment:
Actually the reference information is the search string. Then during export, we retrive and expose the agribalyse identifiers so that people can rely on it to find the used processes.

Thinking about it, I'm not even sure we can define an invariable identifier, because even if the name/unit/location of a given process doesn't change between agribalyse and agribalyse N+1, the upstream processes can be different, which is enough to have a different identifier.  I guess we won't be able unique identifiers on a scope larger than a single database.